npm/private/npm_translate_lock_state.bzl

```diff
@@ -161,9 +161,11 @@ def _init_pnpm_labels(label_store, rctx):
 def _init_update_labels(priv, rctx, label_store):
     attr = rctx.attr
 
+    pnpm_lock_label = label_store.label("pnpm_lock")
+    pnpm_lock_label_str = "//{}:{}".format(pnpm_lock_label.package, pnpm_lock_label.name)
```

Comment:
We need to maintain the same label format in this commit as `consistent_label_str` generated so that our input string to the hash function below doesn't change else users will get a new action cache lock file on this upgrade which is undesirable